src/test/java/com/testCases/UserPostAPI_TCs.java

```diff
@@ -64,10 +64,9 @@ public class UserPostAPI_TCs extends Utilities {
 
     @Test(priority = 1, description = "Create list of users with given array", enabled = true)
```

Comment:
add test cases:
1. add multiple. user in same api hit
2. invalid email id like email@test.
3. phone number greater than 10 digits
4. phone number less than 10 digits
5. validate status for. newly create user
6. with already existing user